### PR TITLE
[HAR-146] Added drop-down menu control

### PR DESCRIPTION
**Pull request by stevengunneweg**

### CODE REVIEW

File: scss/controls/drop-down-menu.scss [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+		background-image: url(get-svg-uri($icon-chevron-down-bold, $color-text-300));
+		background-position: center center;
+		background-size: $spacing * 1.5;
+		background-repeat: no-repeat;
+		content: '';
+		pointer-events: none;
+	}
+}
+
+.drop-down-menu--90 {
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Control Size 90?

File: scss/controls/drop-down-menu.scss [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+		box-shadow: $input-focus-box-shadow;
+	}
+
+	&::-ms-expand {
+		display: none; // NOTE To hide select control in IE10+.
+	}
+
+	&::-ms-value {
+		background: none; // NOTE To remove blue background on IE10+ when selected.
+		color: $color-text; // NOTE To make sure the text is readable on a light background from item-pop-up-menu when selected and input is not checked.
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Comment says pop-up-menu. Should be drop-down-menu option

File: docs/controls/drop-down-menu.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Drop-down menu
+
```
Comment (bartvandebiezen) [inline docs/controls/drop-down-menu.md]: Add a note about the difference between pop-up-menu, drop-down-menu and popover? It is possible Romy already wrote the content. Will send a message with location.

File: scss/controls/drop-down-menu.scss [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+	&.is-invalid {
+	}
+
+	&::after {
+		@include align(absolute, right, center);
+		right: $spacing * 2;
+		display: block;
+		width: $spacing * 2.5;
+		height: $spacing * 2.5;
+		background-image: url(get-svg-uri($icon-chevron-down-bold, $color-text-300));
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Input color or UI color?

File: scss/controls/drop-down-menu.scss [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+		background-position: center center;
+		background-size: $spacing * 1.5;
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Use icon size for width and height and set background size at 100%?

File: scss/controls/drop-down-menu.scss [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+		&:focus {
+			border: $input-is-invalid-focus-border;
+			background-color: $input-is-invalid-focus-background-color;
+			box-shadow: $input-is-invalid-focus-box-shadow;
+		}
+	}
+
+	.drop-down-menu--90 & {
+		padding-right: $spacing * 1.5;
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Should keep space for icon?

File: scss/controls/drop-down-menu.scss [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+		box-shadow: $input-is-invalid-box-shadow;
+
+		&:focus {
+			border: $input-is-invalid-focus-border;
+			background-color: $input-is-invalid-focus-background-color;
+			box-shadow: $input-is-invalid-focus-box-shadow;
+		}
+	}
+
+	.drop-down-menu--90 & {
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Missing new top and bottom padding?

File: scss/controls/drop-down-menu.scss [hunk 8]
```diff
@@ -0,0 +1,152 @@
+.drop-down-menu {
+	position: relative;
+	display: block;
+	width: 100%;
+	height: $control-size-100;
+	color: $input-color;
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Height, color, font size and line height are set in select. Is it necessary on this element?

File: scss/controls/drop-down-menu.scss [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+		&:focus {
+			border: $input-is-invalid-focus-border;
+			background-color: $input-is-invalid-focus-background-color;
+			box-shadow: $input-is-invalid-focus-box-shadow;
+		}
+	}
+
+	.drop-down-menu--90 & {
+		padding-right: $spacing * 1.5;
+		padding-left: $spacing * 1.5;
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Same as 100.

File: scss/controls/drop-down-menu.scss [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+.drop-down-menu--max-width-60 {
+	max-width: $spacing * 16;
+}
+
+.drop-down-menu__select {
+	@include ellipsis();
+	position: relative;
+	display: block;
+	padding-top: (($control-size-100 - ($input-line-height * $input-font-size)) / 2);
+	padding-right: $spacing * 6.25;
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Use same size system as from text input? O.a. “padding-right: $control-size-100;”

File: docs/controls/drop-down-menu.md (hunk 3 above)
Comment (stevengunneweg) [reply docs/controls/drop-down-menu.md]: @bartvandebiezen please provide location :)

File: scss/controls/drop-down-menu.scss [hunk 11]
```diff
@@ -0,0 +1,160 @@
+.drop-down-menu {
+	position: relative;
+	display: block;
+	width: 100%;
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: Is niet nodig

File: scss/controls/drop-down-menu.scss [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+	}
+
+	&.is-invalid {
+	}
+
+	&::after {
+		@include align(absolute, right, center);
+		display: block;
+		width: $control-size-100;
+		height: $icon-size-100;
```
Comment (bartvandebiezen) [inline scss/controls/drop-down-menu.scss]: control-size-100? Een vierkant maken?

**Pull request merged**